src/main/java/org/usfirst/frc4904/robot/commands/FlywheelOff.java

```diff
@@ -0,0 +1,20 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.Flywheel;
+
+public class FlywheelOff extends FlywheelSetSpeed {
+
+  /**
+   * Set flywheel speed to zero
+   * 
+   * @param flywheel The flywheel to manipulate
+   */
+  public FlywheelOff(Flywheel flywheel) {
+    super(flywheel, 0.0);
+  }
+
+  public FlywheelOff() {
+    super(RobotMap.Component.flywheel, 0.0);
```

Comment:
change to `RobotMap.Component.shooter`, `RobotMap.Component.flywheel` has been removed